### DISCUSSION TITLE
uri encode or string replace helper

### DISCUSSION THREAD

**Discussion by c-wygoda** [mockoon/mockoon#789] [Help]
I need to return base64 encoded values as query parameters, therefore the potential padding "=" chars need to be encoded. I can't see an `encodeURIComponent` helper or just a simple string replacement helper.

Am I missing something or is this really missing?

**Comment by 255kb** [comment]
No indeed, there is no URI encode helper. 
Feel free to open an new issue if you find no alternative.
